### PR TITLE
Add service worker

### PR DESCRIPTION
**Pull request by harshita97**
Changes: I have revert the unnecessary changes what else i need to do? And what is this Travis build CI test?

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -74,8 +74,19 @@
 
   <p align="justify"><font size="2px" color="grey"><center>Copyright 2017 &copy; Programming Club,UIET. All Rights Reserved. <br></center></font></p>
 </footer>
-  {% block scripts %}
-  {% endblock %}
+
```
Comment (divyam3897) [inline templates/base.html]: use the script block for scripts.

**Comment by divyam3897** [comment]
@harshita97 are you still working on it? Can you squash your commits into one please.

**Comment by harshita97** [comment]
i tried but i couldn't , I am new at this code contribution. I squashed the changes but still there is 1 more commit which i can not squash with the main commit.

**Comment by divyam3897** [comment]
Firsly remove the merge commits using reset --hard, do a rebase, squash your commits and do push -f

**Comment by divyam3897** [comment]
@harshita97 still working on it?

**Comment by divyam3897** [comment]
Closing due to inactivity

**Pull request closed without merging**